components/Map.tsx

```
@@ -1,33 +1,13 @@
-  if (!apiKey) {
-    console.warn(
-      "EXPO_PUBLIC_GEOAPIFY_API_KEY is not set; StaticMap will not render."
-    );
-    return <View className="w-full h-[300px] rounded-2xl bg-gray-100" />;
-  }
-  const mapUrl = `https://maps.geoapify.com/v1/staticmap?style=osm-bright&width=600&height=400&center=lonlat:${longitude},${latitude}&zoom=${zoom}&apiKey=${apiKey}`;
+import { View, Text } from 'react-native'
+import React from 'react'
+import MapView from 'react-native-maps'
 
+const Map = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)